### PR TITLE
Cs 565 miw ssi agent lib sync the upstream repository

### PR DESCRIPTION
**Pull request by PManaras**

### CODE REVIEW

File: src/main/java/org/eclipse/tractusx/ssi/lib/SsiLibrary.java [hunk 1]
```diff
@@ -27,6 +27,10 @@
 
 /** The type SSI library. */
 public final class SsiLibrary {
+
```
Comment (mustafasalfiti) [inline src/main/java/org/eclipse/tractusx/ssi/lib/SsiLibrary.java:30]: should we change the name to SSILibrary?

File: src/main/java/org/eclipse/tractusx/ssi/lib/crypt/ec/ECKeyGenerator.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
 import org.bouncycastle.jce.provider.BouncyCastleProvider;
 import org.eclipse.tractusx.ssi.lib.crypt.IKeyGenerator;
 import org.eclipse.tractusx.ssi.lib.crypt.KeyPair;
-import org.eclipse.tractusx.ssi.lib.exception.KeyGenerationException;
+import org.eclipse.tractusx.ssi.lib.exception.key.KeyGenerationException;
```
Comment (mustafasalfiti) [inline src/main/java/org/eclipse/tractusx/ssi/lib/crypt/ec/ECKeyGenerator.java:12]: Copywrite should be added to this file

File: src/main/java/org/eclipse/tractusx/ssi/lib/crypt/util/SignerUtil.java [hunk 3]
```diff
@@ -0,0 +1,84 @@
+package org.eclipse.tractusx.ssi.lib.crypt.util;
```
Comment (mustafasalfiti) [inline src/main/java/org/eclipse/tractusx/ssi/lib/crypt/util/SignerUtil.java:1]: Copywrite should be added to this file

**Review by mustafasalfiti — CHANGES_REQUESTED** [review]
Tested with MIW and Revocation Service all Xray tests as well tests are working,
thank you for writting and refactoring some of the tests.

however you are missing the copywrite on the new files i noted some of them but all the new test files are missing that!.

**Review by mustafasalfiti — APPROVED** [review]

**Review by nitin-vavdiya — APPROVED** [review]

**Pull request merged**